### PR TITLE
Feature/nodemailer integration

### PR DESCRIPTION
**Pull request by felmez**

### CODE REVIEW

File: src/utils/mailer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    pass: process.env.EMAIL_PASS,
+  },
+});
+
+const sendEmail = (user) => {
+  const mailOptions = {
+    from: process.env.EMAIL,
+    to: user.email,
+    subject: 'Welcome | Your account created successfully',
+    text: `You can login the to platform by the information below: \n
```
Comment (halitbatur) [inline src/utils/mailer.js:18]: typo

**Review by halitbatur — CHANGES_REQUESTED** [review]
Looks good overall, would like the text to change a little bit there's a typo in the first line and maybe we can use headers, in that case, you can use the `html` key that the mail options provide see more here: https://nodemailer.com/message/

File: src/utils/mailer.js (hunk 1 above)
Comment (felmez) [reply src/utils/mailer.js]: fixed

Comment (felmez) [reply src/utils/mailer.js]: @halitbatur PTAL

File: src/utils/mailer.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const dotenv = require('dotenv');
+
```
Comment (halitbatur) [inline src/utils/mailer.js:2]: can we move the require statements after each other and after it a new line and then `dotenv.config`

**Review by halitbatur — APPROVED** [review]
Looks good only this small change and then feel free to merge.

File: src/utils/mailer.js (hunk 2 above)
Comment (felmez) [reply src/utils/mailer.js:2]: done

File: src/utils/mailer.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+const transporter = nodemailer.createTransport({
+  host: 'smtp.gmail.com',
+  port: 465,
+  auth: {
+    user: process.env.EMAIL,
+    pass: process.env.EMAIL_PASS,
+  },
+});
+
+const sendEmail = (user) => {
```
Comment (Amr-Nash) [inline src/utils/mailer.js:17]: Nice and clean, keep up the good work

**Review by Amr-Nash — APPROVED** [review]

**Pull request merged**